src/rules/selector-max-specificity/README.md

```diff
@@ -14,6 +14,8 @@ This rule ignores selectors with variable interpolation (`#{$var}`, `@{var}`, `$
 
 This rule ignores selectors containing the `:not()` pseudo-class.
 
+This rule ignores selectors containing the `:matches()` pseudo-class.
```

Comment:
Merge into one line .e.g "This rule ignores selectors containing the `:matches()` or `:not()` pseudo-classes."